### PR TITLE
add highlights for 2022-08-25

### PR DESCRIPTION
**Pull request by weatherhog**
This weeks highlights PR

### CODE REVIEW

**Comment by T-Kukawka** [comment]
nothing from Phoenix

File: src/content/changes/highlights/2022-08-25.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+## User interfaces
+
+## Apps
+
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2022-08-25.md:14]:
```suggestion

- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [v2.16.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2160---2022-08-24) updates to upstream controller version [v1.3.0](https://github.com/kubernetes/ingress-nginx/blob/main/Changelog.md#130), removes support for kubernetes v1.19.0 and increases default replica number to 2

```

Comment (stone-z) [inline src/content/changes/highlights/2022-08-25.md:14]:
```suggestion

- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.11.1](https://github.com/giantswarm/kyverno-app/blob/master/CHANGELOG.md#0111---2022-08-23) updates `policy-reporter` and included UIs to version 2.11.1 and sets additional configuration for improving Kyverno's resilience against rate limiting.

```

**Comment by gianfranco-l** [comment]
nothing from honeybadger

**Comment by marians** [comment]
Fishing for an approval here

**Review by marians — APPROVED** [review]

**Comment by marians** [comment]
Ha! Since I'm not the author I can approve myself :)

**Pull request merged**